playground/tests/nuxt/index.spec.ts

```diff
@@ -29,14 +30,15 @@ describe('client-side nuxt features', () => {
   it('defaults to index page', async () => {
     expect(useRoute().matched[0].meta).toMatchInlineSnapshot(`
       {
-        "slug": "foo",
+        "value": "set in index",
       }
     `)
     // TODO: should it be possible to push to other routes?
   })
 
   it('allows pushing to other pages', async () => {
     await useRouter().push('/something')
+    await flushPromises()
```

Comment:
Note: I tried using `await nextTick()` here as well, but since it took multiple of these to properly await for the route to update I think this is a bit cleaner.

Reply:
Hm. It seems we need something a bit more now. 🤔

Reply:
There is some precedence in the `vue-router` tests to use double `tick()` (although they use `process.nextTick()`) to await some certain state.

I suppose with e.g. guards there could be many ticks when it comes to awaiting a specific route state. The most robust solution is probably an async matcher that checks on an interval, like [wait-for-expect](https://github.com/TheBrainFamily/wait-for-expect).

For simplicity I've updated the PR to use double `nextTick()` from vue. Lets see if that goes through CI.